products.html

```diff
@@ -0,0 +1,116 @@
+            <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
+        </div>
+        <div class="text-container">
+            <h2>The</h2>
+            <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
+        </div>
+        <div class="text-container">
+            <h2>Stuff</h2>
+            <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
+        </div>
+
+    </section>
+        
+    <section class="callToAct">
+        <h2>
+            Why our products?
+        </h2>
+        <P>
+          Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.        
+        </P>
+      </section>
+
+        <section class="middle-content">
+        
+            <h2>Our Products</h2>
+            
+            <div class="boxes">
+                <div class="box box1">Cat Food</div>
+                <div class="box box3">DnD Mini</div>
+                <div class="box box1">1984</div>
+                <div class="box box3">Pencils</div>
+                <div class="box box1">Mad Vidz</div>
+                <div class="box box5">Lord</div>
+                <div class="box box7">Of</div>
+                <div class="box box5">The</div>
+                <div class="box box7">Rings</div>
+                <div class="box box5">Diapers</div>
+            </div>
+        
+        </section>
+
+        <section class="newBoxes">
+            <div class="box">
+                <img src="img/about-plan.png" alt="strategy">
+              <h2>
+                  Buy Direct
+              </h2>
+              <p>
+                  Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+              </p>
+              <button>
+                Learn More
+              </button>
+            </div>
+        
+            <div class="box">
+                <img src="img/about-meeting.png" alt="strategy">
+                <h2>
+                    Find a retailer near you
+                </h2>
+                <p>
+                  Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+              </p>                  <button>
+                    Learn More
+                </button>
+              </div>
+      </section>
+
+        <footer>
+            <nav>
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
```

Comment:
It's easy to forget to put links in your footer navs. When you think your work is done, a good habit is to just go around and click everything that can be clicked, to see if anything's broken or not wired up properly. Your pages look excellent, and this was the only "flaw" I could detect! Not really a flaw, just a _very minor_ oversight.